1)  Take 5 different distance measurements noting the exact distance.  Report these values. 
---

#1 had at 30 cm got 29.50

#2 hand at 21 cm got 20.72

#3 hand at 5 inch got 12.9

#4 hand at 6 inch got 15.14

#5 hand at 8 inch got 20.30

2) Calculate how many microseconds one foot is.  Show your work
---


we can use the formula (speed of sound in feet * time in seconds) / 2<br>
we can then do some algabra to find the answer<br>
$1 = (1125.33 * time in second) / 2$<br>
solving we for time in second we get<br>
$\frac{1}{2} = (1125.33 * t)$<br>
$ \frac{\frac{1}{2}}{1125.33} = t$



In [ ]:
(1/2)/1125.33

0.00044431411230483504

so it takes around 0.00044431411230483504 seconds for sound to travel 1 foot<br>
we then can convert that into microseconds

In [ ]:
0.00044431411230483504 * 1000000

444.314112304835

so the answer is it takes 444.314112304835 microseconds for sound to travel one foot

3)  What is the minimum and maximum effective measuring distance of the device?  Provide a link along with your answer
---

https://docs.arduino.cc/built-in-examples/sensors/Ping/

from 3 cm up to 400 cm

4)  Is it possible for our ISR to be interrupted with another PINB, pin 1 state change?
---

i don't think so the i bit must be hi in other words the interrupts must be on. <br>
the i don't know if the i bit is on or off.<br>